### ISSUE TITLE
Teleoperation PSM jaw/gripper offset not preserved

### ISSUE THREAD

**Issue by adeguet1** [jhu-dvrk/sawIntuitiveResearchKit#122]
Maintain error offset between jaws and gripper to avoid jump when teleoperation enters follow mode.   This should be similar to the alignment offset.  Add ROS topic to publish said offset.
